src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/CourtsService.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.repository.CourtRepository
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.mapping.toModel
+
+@Service
+class CourtsService(private val courtRepository: CourtRepository) {
```

Comment:
Courts service or ReferenceDataService?  Dunno just a thought.

Reply:
I did wonder about combining them, but then thought there is likely to be more court or probation team specific endpoints to come, so separated.